src/analytics/internal/analytics.js

```diff
@@ -0,0 +1,156 @@
+import { MapEventTypeToInt, MapNotifTypeToIntArray } from './constants'
+import db from 'src/search/search-index-new'
+import AnalyticsStorage from './AnalyticsStorage'
+import SendToServer from './sendToServer'
+
+class Analytics {
+    _queueEvents = []
+    _loaded = 0
+    _operationsMap = {
```

Comment:
This state keeps track of the event count and last time in mem, which is nice, but I can't see it being used anywhere apart from `getCountNotif` method, although that isn't called from anywhere?

Reply:
We will use these things in activity based notifications. We don't want to query every time. Therefore we are having all the notif type in memory. Make sense?

Reply:
I understood it's purpose, but didn't understand why it wasn't used anywhere. Makes sense if the activity-based notifs will be implemented later 👍